buku

```diff
@@ -3570,7 +3570,7 @@ def import_html(html_soup: BeautifulSoup, add_parent_folder_as_tag: bool, newtag
         comment_tag = tag.findNextSibling('dd')
 
         if comment_tag:
-            desc = comment_tag.find(text=True, recursive=False)
+            desc = comment_tag.find(string=True, recursive=False)
```

Comment:
`text` parameter was renamed to `string` in 4.4.0 [due to confusing semantics](https://bugs.launchpad.net/beautifulsoup/+bug/1366856).

The deprecation warning was added in 4.11.0 [to prevent usage of `text` as a keyword parameter](https://bugs.launchpad.net/beautifulsoup/+bug/1947038).